cmd/istioctl/istioctl_test.go

```diff
@@ -155,3 +155,19 @@ func TestNewGet(t *testing.T) {
 		t.Fatalf("Could not delete world-cb: %v", err)
 	}
 }
+
+func TestInvalidRouteRule(t *testing.T) {
+	rootSetup(t)
+	file = "testdata/invalid-route-rule.yaml"
+	if err := postCmd.RunE(postCmd, []string{}); err == nil {
+		t.Fatalf("Did not fail when presented with invalid route rule")
+	}
+}
+
+func TestInvalidDestinationPolicy(t *testing.T) {
+	rootSetup(t)
+	file = "testdata/invalid-destination-policy2.yaml"
+	if err := postCmd.RunE(postCmd, []string{}); err == nil {
```

Comment:
Please sort out some sane naming for the files. And please keep it consistent as well. Whether it's *policy1.yaml, *policy2.yaml, etc or something else. As long as the names follow a consistent pattern, it's fine